test/index.js

```diff
@@ -466,4 +466,78 @@ describe('Hapi csv', () => {
             });
         });
     });
+
+    describe('Result key (e.g. for pagination)', () => {
```

Comment:
I would like to also see a test case with the resultKey option where the result does not have the key, just for good measure.